### PR TITLE
[IN-35] Share a link to a specific NFT as a MINTER

### PR DESCRIPTION
**Pull request by Prajay-Chaudhary**
Linked to this ticket: https://instamint-deva.atlassian.net/browse/IN-35

### CODE REVIEW

File: src/components/custom/feed/share-specific-nft-link/share-button.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  const locale = useLocale()
+  const origin = typeof window !== "undefined" ? window.location.origin : ""
+  const localizedUrl = locale ? `${origin}/${locale}${url}` : `${origin}${url}`
+  const handleShareClick = () => {
+    navigator.clipboard.writeText(localizedUrl).then(
+      () => {
+        toast.success(t("successUrlLink"))
+      },
+      () => {
+        toast.success(t("errorCopyMessage"))
```
Comment (jimmy-martin) [inline src/components/custom/feed/share-specific-nft-link/share-button.tsx]: as it is an error, why do you use `toast.success` ?

File: src/components/custom/feed/share-specific-nft-link/share-button.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import { LuSend } from "react-icons/lu"
+import { RWebShare } from "react-web-share"
+import { toast } from "sonner"
+
+export default function ShareButton({ title, text, url, iconSize = 24 }: ShareButtonProps) {
+  const t = useTranslations("shareNftLink")
+  const locale = useLocale()
+  const origin = typeof window !== "undefined" ? window.location.origin : ""
+  const localizedUrl = locale ? `${origin}/${locale}${url}` : `${origin}${url}`
+  const handleShareClick = () => {
```
Comment (jimmy-martin) [inline src/components/custom/feed/share-specific-nft-link/share-button.tsx]: For handlers we use to use `useCallback`

**Review by jimmy-martin — CHANGES_REQUESTED** [review]
As it is a share button, maybe you can add one inside an NFT page and in search NFT card too (or you can create a ticket to do it after if you have more important things to develop for presentation).

File: src/components/custom/feed/share-specific-nft-link/share-button.tsx (hunk 1 above)
Comment (Prajay-Chaudhary) [reply src/components/custom/feed/share-specific-nft-link/share-button.tsx]: My bad.

File: src/components/custom/feed/share-specific-nft-link/share-button.tsx (hunk 2 above)
Comment (Prajay-Chaudhary) [reply src/components/custom/feed/share-specific-nft-link/share-button.tsx]: Yes, you're right. I totally forgot to use it.

**Comment by Prajay-Chaudhary** [comment]
> As it is a share button, maybe you can add one inside an NFT page and in search NFT card too (or you can create a ticket to do it after if you have more important things to develop for presentation).

Done. Since it's just about adding the existing component inside an NFT page and in search NFT card, we don't need a new PR to do  it.

File: src/components/custom/feed/share-specific-nft-link/share-button.tsx (hunk 1 above)
Comment (Prajay-Chaudhary) [reply src/components/custom/feed/share-specific-nft-link/share-button.tsx]: Corrected.

**Review by jimmy-martin — APPROVED** [review]

**Pull request merged**